graph/src/log/mod.rs

```diff
@@ -53,6 +53,10 @@ pub fn logger(show_debug: bool) -> Logger {
     Logger::root(drain, o!())
 }
 
+pub fn discard() -> Logger {
+    Logger::root(slog::Discard, o!())
```

Comment:
Dumb question, why/what is this for? 🤔

Reply:
Convenience to construct a logger that discards messages.